tests/operators/test_trigger_dagrun.py

```diff
@@ -434,7 +445,18 @@ def test_trigger_dagrun_with_wait_for_completion_true_defer_true_failure(self):
             assert len(dagruns) == 1
 
         with pytest.raises(AirflowException):
```

Comment:
Since there are more than one AirflowException that can be thrown, it might be a good idea to specify add a `match` here to make sure the proper AirflowException is seen. WDYT?